testing/go/update_test.go

```diff
@@ -94,5 +94,44 @@ func TestUpdate(t *testing.T) {
 				},
 			},
 		},
+		{
+			Name: "UPDATE ... RETURNING",
+			SetUpScript: []string{
+				"CREATE TABLE t (pk INT PRIMARY KEY, c1 TEXT);",
+				"INSERT INTO t VALUES (1, 'one');",
+			},
+			Assertions: []ScriptTestAssertion{
+				{
+					Query:    "UPDATE t SET c1 = '42' RETURNING c1;",
```

Comment:
Would have a test here where the update is an expression instead of al iteral, eg `set pk = pk+1`

Reply:
Also a test of returning more than a single column